### PR TITLE
Remove image endpoints and default originCode

### LINKED ISSUE #793: Remove the image base endpoints from fixture data
**Is your feature request related to a problem? Please describe.**
We currently have image locators such as `http://c.files.bbci.co.uk/E7DB/production/_101655395_paulineclayton.jpg` however Optimo will be delivering these as just locator `E7DB/production/_101655395_paulineclayton.jpg`.

**Describe the solution you'd like**
We need to remove the endpoint prefix from the image locators

**Testing notes**
Images should still render, visual regression and both unit and cypress tests 

**Additional context**
This issue can be done in conjunction with https://github.com/BBC-News/simorgh/issues/786

- [x] Intially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #793 

_This PR removes the base path from the image URLs in the fixture data, this aligns with the Optimo data as it only supplies locators. This PR also refactors the originCode [logic](https://github.com/BBC-News/simorgh/pull/798/files#diff-445cfd09f5ef4bfbe3544be7b7f2da00R25) so that null or empty string origin codes default to `cpsdevpb` this is needed because Optimo currently does not supply an originCode and the work is not yet scoped. As part of the originCode refactor I ensured the requirement to return the locator value for the originCode of `pips` was still met._

- [x] Tests added for new features
- [ ] Test engineer approval

## Test instructions ##
Unit test and Cypress tests should cover these changes, I also suggest a basic manual regression to ensure images load

### CODE REVIEW

File: src/app/containers/Image/index.jsx [hunk 1]
```diff
@@ -22,13 +22,18 @@ const getCopyright = copyrightHolder => {
...
-  `https://ichef.bbci.co.uk/news/${DEFAULT_IMAGE_RES}/${originCode}/${
-    locator.split('http://c.files.bbci.co.uk/')[1]
-  }`;
+const getIChefURL = (originCode, locator) => {
+  let overridableOriginCode = originCode;
+  // temp code - default to 'cpsdevpb' until Optimo complete work to supply non-empty originCode
+  if (!overridableOriginCode) {
+    overridableOriginCode = 'cpsdevpb';
+  }
+
```
Comment (radiocontrolled) [inline src/app/containers/Image/index.jsx:28]: I think you can do: 
` const overridableOriginCode = originCode || 'cpsdevpb';`

This allows avoidance of using `let`

**Review by radiocontrolled — CHANGES_REQUESTED** [review]
LGTM -I  like the improved test descriptions

I approve pending a change to not use `let` which I have requested a change for.

File: src/app/containers/Image/index.jsx (hunk 1 above)
Comment (pjlee11) [reply src/app/containers/Image/index.jsx:28]: Good point, will change

**Review by radiocontrolled — APPROVED** [review]
⚽️

**Review by jtart — APPROVED** [review]

**Pull request merged**